src/object-store/tests/parser.cpp

```diff
@@ -34,10 +38,13 @@ static std::vector<std::string> valid_queries = {
     // operators
     "0=0",
     "0 = 0",
+    "0 =[c] 0",
     "0!=0",
     "0 != 0",
     "0==0",
     "0 == 0",
+    "0==[c]0",
+    "0 == [c] 0",
```

Comment:
Like I said above, since we're using `NSPredicate` syntax, we shouldn't arbitrarily add whitespace restrictions to it.